src/pages/[platform]/build-a-backend/auth/concepts/multi-factor-authentication/index.mdx

```diff
@@ -28,11 +28,15 @@ export function getStaticProps() {
   };
 }
 
-Amplify Auth supports Multi-factor Authentication (MFA) for user sign-in flows. MFA is an extra layer of security used to make sure that users trying to gain access to an account are who they say they are. It requires users to provide additional information to verify their identity. Amplify Auth supports the MFA methods with Time-based-One-Time Passwords (TOTP) as well as text messages (SMS). In this guide we will review how you can set up MFA using TOTP and SMS and the tradeoffs between these methods to help you choose the right set up for your application. We will also review how to set up MFA to remember a device and reduce sign-in friction for your users.
+Amplify Auth supports multi-factor authentication (MFA) for user sign-in flows. MFA is an extra layer of security used to make sure that users trying to gain access to an account are who they say they are. It requires users to provide additional information to verify their identity. Amplify Auth supports MFA with time-based one-time passwords (TOTP), text messages (SMS), and email. 
+
+In this guide we will review how you can set up MFA with each of these methods and the discuss tradeoffs between them to help you choose the right setup for your application. We will also review how to set up MFA to remember a device and reduce sign-in friction for your users.
 
 ## Configure multi-factor authentication
 
-Use `defineAuth` to enable MFA for your app. The example below is setting up MFA with TOTP but not SMS as you can see that the phone number is not a required attribute. If you are using SMS, then the `PhoneNumber` attribute must be `true`.
+Use `defineAuth` to enable MFA for your app. The example below is setting up MFA with TOTP but not SMS as you can see that the phone number is not a required attribute. 
+- If you plan to use SMS for MFA, then the `phoneNumber` attribute must be `true`. 
+- If you plan to use email for MFA, then the `email` attribute must also be `true`.
```

Comment:
do these need to be enabled as username attributes or do they just need to be marked as "required" in the user attributes?

Reply:
that's a more accurate way to phrase it, updated [here](https://github.com/aws-amplify/docs/pull/8023/commits/205d43eaafff6bed52e8546f95e921fbb91b28fb)